components/selection/selection-controls.js

```diff
@@ -150,8 +155,11 @@ export class SelectionControls extends PageableSubscriberMixin(SelectionObserver
 		if (changedProperties.has('noSticky')) {
 			this._stickyObserverUpdate();
 		}
-		if (changedProperties.has('_pageableInfo')) {
-			this._noSelectionText = this._getNoSelectionText();
+	}
+
+	willUpdate(changedProperties) {
+		if (changedProperties.has('noSelectionText') || changedProperties.has('_pageableInfo')) {
+			this._noSelectionText = this.noSelectionText ? this.noSelectionText : this._getNoSelectionText();
```

Comment:
Could probably simplify with `this.noSelectionText || this._getNoSelectionText()`.